### PR TITLE
Clean up InitPlayerGFX

### PR DESCRIPTION
**Pull request by AJenbo**

### CODE REVIEW

File: Source/player.cpp [hunk 1]
```diff
@@ -280,22 +280,16 @@ void __fastcall LoadPlrGFX(int pnum, int gfxflag)
...
-		v2 = 0x17F;
 	}
-	else
-	{
-		plr[v1]._pgfxnum = 0;
-		v2 = 0x80;
+
+	if ( !(plr[pnum]._pHitPoints & 0xFFFFFFC0) ) {
+		plr[pnum]._pgfxnum = 0;
+        LoadPlrGFX(pnum, 0x80);
```
Comment (seritools) [inline Source/player.cpp]: weird indentation over here

File: Source/player.cpp [hunk 2]
```diff
@@ -280,22 +280,16 @@ void __fastcall LoadPlrGFX(int pnum, int gfxflag)
...
-	if ( plr[v1]._pHitPoints & 0xFFFFFFC0 )
-	{
-		v2 = 0x17F;
 	}
-	else
-	{
-		plr[v1]._pgfxnum = 0;
-		v2 = 0x80;
+
+	if ( !(plr[pnum]._pHitPoints & 0xFFFFFFC0) ) {
```
[the remark this answers is not in the document]
Comment (mewmew) [reply Source/player.cpp]: Perhaps even `plr[pnum]._pHitPoints/64 == 0`, as the compiler will perform [strength reduction](https://en.wikipedia.org/wiki/Strength_reduction#Other_strength_reduction_operations) optimization.

File: Source/player.cpp (hunk 1 above)
Comment (mewmew) [reply Source/player.cpp]: We should really try to get https://github.com/diasurgical/devilution/issues/182 resolved, as we can then ignore whitespace diffs.

File: Source/player.cpp (hunk 2 above)
[the remark this answers is not in the document]
Comment (seritools) [reply Source/player.cpp]: @mewmew depends on if the number is signed or not i think. But definitely worth a try! :)

File: Source/player.cpp (hunk 1 above)
Comment (AJenbo) [reply Source/player.cpp]: I think it can be done gradually, I'll set up something and du a pr

**Comment by AJenbo** [comment]
Updated with suggestions. Is it the plan to change things to DWORD eventually so that we avoid all the casting in the code?

**Comment by seritools** [comment]
Not in this case, these `pnum` comparison casts are literally in the original code.

**Pull request merged**